### DISCUSSION TITLE
Any plans to support WiFi 6 dongles?

### DISCUSSION THREAD

**Discussion by webhostsg** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#229] [General]
Been using Hackintosh for several years, but always connected through an ethernet cable. My current ISP provided a WiFi 6 mesh, I have the satellite right beside my Hackintosh and connected by wire to it. But now I need to move the satellite to the kitchen, the router is too far to wire it directly. Was thinking would buying a WiFi 6 USB dongle work. I have OpenIntel Wireless on my NUC8 Hackintosh but the speeds are just too slow only about 50mbps (mainly the upload). I need to upload large files regularly for work.

**Comment by chris1111** [comment]
This utility will not be update of anny, the wifi companies have abandoned the macOS project

**Reply by webhostsg** [reply]
> This utility will not be update of anny, the wifi companies have abandoned the macOS project

Ok, noted. Thank you for the quick reply.
